config/manager/kustomization.yaml

```diff
@@ -12,6 +12,6 @@ configMapGenerator:
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
-- name: controller
+- name: quay.io/eclipse/che-operator:next
```

Comment:
hmm Does it really need?

Reply:
Yes, it's how this kustomize feature works.